CONTRIBUTING.md

```diff
@@ -1,35 +1,211 @@
+    -   Windows: `%LocalAppData%\qBittorrent\logs`
+    -   macOS: `~/Library/Application Support/qBittorrent/qBittorrent.log`
+
+
+-   **Do NOT post comments like "+1" or "me too!"** without providing new relevant info on the issue. Using the built-in reactions is OK though.
+
+-   If there seems to be an **issue with specific torrent files/magnet links**:
+    -   Always include links to the specific torrent(s)/magnet(s) where the issue occurs, even if "it happens everywhere".
+    -   Make sure you can't the problem with another client, to rule out the possibility that the issue is with the torrent file/magnet link itself.
+
+
+-   A screenshot, transcription or file upload of any of **qBittorrent's preferences that differ from the defaults**. Please include everything different from the defaults whether or not it seems relevant to your issue.
+
+-   **Attachment rules**:
+    -   Short logs and error messages can be pasted as quotes/code whenever small enough; otherwise make a gist with the contents and post the link to the gist;
+    -   Avoid linking/attaching impractical file formats such as PDFs/Word documents with images. If you want to post an image, just post the image.
+
+
+
+### Provide more context by answering these questions (if applicable):
+
+-   Are you **using qBittorrent with multiple monitors?** If so, can you reproduce the problem when you use a single monitor?
+
+-   Can you **reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens (e.g. only happens with extremely large torrents/only happens after qBittorrent is open for more than 2 days/etc...)
+
+-   Did the problem start happening recently (e.g. after updating to a new version of qBittorrent) or was this always a problem?
+
+-   If the problem started happening recently, can you reproduce the problem in an older version of qBittorrent? What's the most recent version in which the problem doesn't happen?
+
+-   Are you saving files locally (in a disk in your machine), or are you saving them remotely (e.g. network drives)?
+
+Good read: [How to Report Bugs Effectively](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
+
+
+# Suggesting Enhancements/feature requests
+
+This section guides you through submitting an enhancement suggestion for qBittorrent, including completely new features and minor improvements to existing functionality.
+
+Following these guidelines helps maintainers and the community understand your suggestion and find related suggestions.
+
+
+## Before submitting an enhancement/feature request
+
+*   Check the wiki and forum for tips — you might discover that the enhancement is already available.
+*   Most importantly, check if you're using the latest version of qBittorrent and if you can get the desired behavior by changing qBittorrent's settings.
+*   Check if there's already a beta version that enhancement.
```

Comment:
there is usually not a beta around.
or say: Check on the forum, see if there's already a beta version with that enhancement.